src/evaluations/report_generator.py

```diff
@@ -0,0 +1,286 @@
+# Copyright 2020 The Private Cardinality Estimation Framework Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Generate evaluation report."""
+
+import os
+import shutil
+
+import pandas as pd
+
+from wfa_cardinality_estimation_evaluation_framework.evaluations import analyzer
+from wfa_cardinality_estimation_evaluation_framework.evaluations import evaluator
+from wfa_cardinality_estimation_evaluation_framework.evaluations.analyzer import KEY_DESCRIPTION_TO_FILE_DIR
+from wfa_cardinality_estimation_evaluation_framework.evaluations.analyzer import KEY_NUM_ESTIMABLE_SETS_STATS_DF
+from wfa_cardinality_estimation_evaluation_framework.evaluations.analyzer import KEY_RUNNING_TIME_DF
+from wfa_cardinality_estimation_evaluation_framework.evaluations.data import evaluation_configs
```

Comment:
data is not part of the package. This file needs to be moved out into the top-level evaluations/ directory.
(or the whole evaluations folder should be in src/. Either one will do.)

Reply:
done, and confirmed run_evaluation works from git clone (although I didn't set the required flags so it didn't fully run, just was able to import things)